core/server/api/v2/utils/serializers/input/posts.js

```diff
@@ -2,6 +2,14 @@ const _ = require('lodash');
 const debug = require('ghost-ignition').debug('api:v2:utils:serializers:input:posts');
 const url = require('./utils/url');
 
+function removeMobiledocFormat(frame) {
+    if (_.get(frame, 'options.formats') && _.get(frame, 'options.formats').includes('mobiledoc')) {
```

Comment:
This can reuse `isContentAPI` method from PR [here](https://github.com/TryGhost/Ghost/pull/10095)

Reply:
@gargol @rishabhgrg Please sync about this helper. IMO the helper should be pushed to master and then you can both rebase 👍